packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,42 +5,54 @@ keywords: Request Client, Payment networks
 
 ---
 
-# TODO
+When a user creates and sends a request, he can expect to receive the correct amount of money. But how does he keep track of the payments due and received? Request is blockchain agnostic, Request is, by itself, a ledger that documents request's information but it doesn't interact directly with any external blockchains.
 
-When a user creates and sends a request, he can expect to receive the correct amount of money. But how does he keep track of the payments due and received?
+There are different methods available for the requester and payer to agree on the payment status, and that is when payment networks come into play. A payment network is a specific context on how to agree on the payment status of a specific request.
 
-There are different methods available for the requester and payer to agree on the payment status, and that is when Payment networks come into play.
+A payment network is defined by:
+* An extension in the Request protocol that defines the necessary information to be able to detect payments
+* A payment method, it's the method used to perform a detectable payment
+* A payment detection method, it's the process to determine the amount paid by the payer through the payment method
```

Comment:
I honestly think this doesn't help understanding a payment network. 
Instead, I would list and explain 
1. types of detection: 
- address based
- proxy contract
- declarative

2. which currencies are supported by which networks

Reply:
I agree with @benjlevesque, it is too much detailed for this section. 

But it is good explanation tho: this should be kept in another section like "Protocol in depth".

Reply:
@benjlevesque 
I add a section to explain the current type a Payment Network but I think it's better to give some theory instead of giving examples directly.

@vrolland  "Too much detailed for this section"
This page is precisely created to explain what is a Payment Network, IMO it should be detailed.